internal/querycoord/global_meta_broker.go

```diff
@@ -68,7 +68,7 @@ func (broker *globalMetaBroker) releaseDQLMessageStream(ctx context.Context, col
 		log.Error("releaseDQLMessageStream occur error", zap.Int64("collectionID", collectionID), zap.Error(err))
 		return err
 	}
-	log.Debug("releaseDQLMessageStream successfully", zap.Int64("collectionID", collectionID))
+	log.Info("releaseDQLMessageStream successfully", zap.Int64("collectionID", collectionID))
```

Comment:
If these logs are for developer, maybe we should keep it debug level.

Reply:
Same problem occurs in other functions.

Reply:
since this is control path, the log won't be too much~
Otherwise if user set log to info it is very hard to debug